components/compliance-service/api/common/common.proto

```diff
@@ -4,12 +4,17 @@ package chef.automate.domain.compliance.api.common;
 option go_package = "github.com/chef/automate/components/compliance-service/api/common";
 
 message Filter {
+	// Field to filter on.
```

Comment:
```suggestion
	// Applied filter.
```

Reply:
it's the filter the user wants to apply to the query, part of the request object -- should that be "applied filter"?

Reply:
OK